blueprints/in-repo-engine/routable-files/lib/__name__/addon/templates/application.hbs

```diff
@@ -1 +1 @@
-{{outlet}}
+{{outlet}}
```

Comment:
The default ember-cli blueprint prefers no end newlines in templates
https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/.editorconfig#L17